lexicons/com/atproto/sync/subscribeRepos.json

```diff
@@ -105,6 +105,10 @@
             "description": "List of new blobs (by CID) referenced by records in this commit."
           }
         },
+        "prevData": {
```

Comment:
Spoke about this possibly being the right moment to finalize deprecation of `prev` by removing it, also disambiguating `prevData`.